test/tests/functional/pbs_preemption.py

```diff
@@ -54,6 +54,10 @@ def setUp(self):
              'enabled': 'True',
              'Priority': 200}
         self.server.manager(MGR_CMD_CREATE, QUEUE, a, "expressq")
+        if len(self.moms) == 2:
+            self.mom1 = self.moms.keys()[0]
+            self.mom2 = self.moms.keys()[1]
+            self.server.manager(MGR_CMD_DELETE, NODE, id=self.mom2)
```

Comment:
can you please add a comment here to explain why you are deleting the nodes on mom2 ?